src/components/marker-table/index.js

```diff
@@ -130,7 +130,7 @@ class MarkerTree {
             break;
           case 'Text':
             category = 'Text';
-            name += ` [${data.name}]`;
+            name += ` — [${data.name}]`;
```

Comment:
I think we should also remove the brackets :)

Reply:
Done! and commit [e7a6406](https://github.com/firefox-devtools/profiler/pull/1906/commits/e7a64067b00a49a7f5d1f93a8045add981e70002) shows what i mean about the space after `FilterMarker: `.